lib/engines/content_block_manager/config/routes.rb

```diff
@@ -12,6 +12,7 @@
       resources :editions, only: %i[new create destroy], path_names: { new: ":block_type/new" } do
         member do
           resources :workflow, only: %i[show update], controller: "editions/workflow", param: :step
+          resources :preview, only: %i[show], controller: "editions/preview", param: :host_content_id
```

Comment:
hmm my only thinking is that this will eventually be getting a preview for a specific content block edition - so by having it be `/content-block/editions/:edition_id/preview/:uuid` we get the content block info 'for free' from its id. But I guess we could pass the content block information as a query param? 
Or is it the `host_content` you think it should have, so could be `/content-block/editions/:edition_id/host_content/:host_id/preview/`?

Reply:
Oh yeah, good point - `/content-block/editions/:edition_id/host_content/:host_id/preview/` is better in this case I think

Reply:
done 👍